src/main/java/io/meeds/qa/ui/pages/page/factory/space/SpaceHomePage.java

```diff
@@ -1215,4 +1214,40 @@ public void viewAllRepliesInCommentsDrawer(String comment) {
     getCommentsDrawerViewAllReplies(comment).clickOnElement();
   }
 
+  private BaseElementFacade getReactionActivityList(String activity) {
+    return findByXPathOrCSS(String
+            .format("(//*[contains(text(),'%s')]//following::p[contains(@class,'likersNumber')])[1]", activity));
+  }
+
+  public void clickReactionList(String activity) {
+    getReactionActivityList(activity).clickOnElement();
+  }
+
+  @FindBy(xpath = "//*[contains(@class,'v-list-item__content drawerTitle') and contains (text(),'Reactions')]")
+  private BaseElementFacade reactionDrawer;
+
+  public void drawerDisplay() {
+    reactionDrawer.shouldBeVisible();
+  }
+
```

Comment:
```suggestion
```

No need to check on this, please use the generic method `waitForDrawerToOpen()` to wait on drawer to be opened before proceeding on next checks which will have to search for liker names inside the drawer.
In fact, the automatic test has to be performant as much as to be pertinent and efficient while testing.

Reply:
function is updated